chromium/rules.js

```diff
@@ -337,3 +324,7 @@ RuleSets.prototype = {
     return null;
   }
 };
+
+// Export for HTTPS Rewriter if applicable.
+if (typeof exports != 'undefined')
+  exports.RuleSets = RuleSets;
```

Comment:
I really hate bracket-less if statements. But keep it if that's the local style.